pilot/pkg/bootstrap/server.go

```diff
@@ -223,7 +223,7 @@ func NewServer(args *PilotArgs) (*Server, error) {
 	} else {
 		args.RegistryOptions.KubeOptions.EndpointMode = kubecontroller.EndpointsOnly
 	}
-
+	args.SecuredDebug = features.SecuredDebug.Get()
```

Comment:
I am not sure why/if we need `features.SecuredDebug` at all. Isn't this already controllable via https://github.com/istio/istio/blob/88a2bfb3bd8e0010229ee66befc4e56b326c2a3a/pilot/cmd/pilot-discovery/main.go#L137 and we just need to change the default and handle the multiplexing behavior?

Reply:
How would user change the default behavior?   https://github.com/istio/istio/blob/master/manifests/charts/istio-control/istio-discovery/templates/deployment.yaml#L75  -- add a line similar with this will solve this?

Reply:
I think they can already overwrite the `args`: https://preliminary.istio.io/latest/docs/setup/install/istioctl/#patching-the-output-manifest. We don't need to change manifests/ at all probably

Reply:
debug on localhost should be enough for most cases.  So let's skip changes in manifests/.